### PR TITLE
Delete repeated words and fix misspellings

### PR DESCRIPTION
**Pull request by xichengliudui**
**What this PR does / why we need it**:
Delete repeated words and fix misspellings

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

**Special notes for your reviewer**:

**Release note**:
```
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/util/config/initconfiguration.go [hunk 1]
```diff
@@ -118,7 +118,7 @@ func SetAPIEndpointDynamicDefaults(cfg *kubeadmapi.APIEndpoint) error {
 	return nil
 }
 
-// SetClusterDynamicDefaults checks and sets configuration values for the InitConfiguration object
+// SetClusterDynamicDefaults checks and sets ClusterConfiguration values for the InitConfiguration object
 func SetClusterDynamicDefaults(cfg *kubeadmapi.ClusterConfiguration, advertiseAddress string, bindPort int32) error {
```
Comment (neolit123) [inline cmd/kubeadm/app/util/config/initconfiguration.go]: should be:
> SetClusterDynamicDefaults checks and sets values for the ClusterConfiguration object

**Comment by fabriziopandini** [comment]
@xichengliudui Thanks for this cleanup! @neolit123, @mcrute thank for helping in getting this ready for merge
/approve
/lgtm

**Pull request merged**